creator-node/src/services/stateMachineManager/stateMachineConstants.js

```diff
@@ -82,9 +100,19 @@ module.exports = {
       value: 4
     }
   }),
+
   // Describes the type of sync operation
   SyncType: Object.freeze({
     Recurring: 'RECURRING', // Scheduled background sync to keep secondaries up to date
     Manual: 'MANUAL' // Triggered by a user data write to primary
-  })
+  }),
+
+  // Sync mode for a (primary, secondary) pair for a user
+  SYNC_MODES: Object.freeze({
+    None: 'NONE',
+    SecondaryShouldSync: 'SECONDARY_SHOULD_SYNC',
+    PrimaryShouldSync: 'PRIMARY_SHOULD_SYNC'
```

Comment:
nit: it'd be a easier to understand if it were more like "SYNC_SECONDARY_TO_PRIMARY" or "FROM_PRIMARY_TO_SECONDARY"

Reply:
you right, will change